packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -10,6 +10,7 @@ import * as fs from 'fs';
 import { EOL } from 'os';
 import * as path from 'path';
 import { mkdir, rm } from 'shelljs';
+import * as minSObjectsFromFile from '../../minSObjects.json';
```

Comment:
Does this work when using the vsix files ? I think we need to change it to something like we do here https://github.com/forcedotcom/salesforcedx-vscode/blob/93ca07ba5ef8f2a20f6431a5f7075a6637658d43/packages/salesforcedx-vscode-lwc/src/metasupport/metaSupport.ts#L32

Reply:
I haven't tested with vsix yet. Will try this.

Reply:
Actually I'm wrong, this should work cause you're reading it on the faux-generator module side not from the code directly in the `salesforcedx-vscode-*` modules. However, we might want to move it somewhere under the `src` folder.

Reply:
@lcampos - How about moving it to a new "data" directory? It will align with where we have registry.json in SDR.

Reply:
That works too 👍

Reply:
+1 for under `src`. `src/data` is fine too :)